cypress.config.js

```diff
@@ -1,7 +1,8 @@
-const {defineConfig} = require('cypress');
-const {tagify} = require('cypress-tags');
+const { defineConfig } = require('cypress');
+const { tagify } = require('cypress-tags');
```

Comment:
Could you remove the spaces here? The coding standard requires there are no spaces there.

Reply:
Hi @peterjaap , can you please take a look now? I have removed the whitespaces.